packages/codec/lib/abify.ts

```diff
@@ -250,19 +287,24 @@ export function abifyResult(result: Values.Result, userDefinedTypes?: Types.Type
  * but you can, uh, throw an exception if you don't, I guess.)
  */
 
-export function abifyCalldataDecoding(decoding: CalldataDecoding, userDefinedTypes: Types.TypesById): CalldataDecoding {
-  if(decoding.decodingMode === "abi") {
+/** @category ABIfication */
```

Comment:
I can't add a comment on something that hasn't changed, so let me note that the comment above is out of date; these functions are used now.

Reply:
Cool I will remove